File: Java/Aksesoris.java
class Aksesoris extends Petshop {
    private String jenis;
    private String bahan;
    private String warna;
    
    public Aksesoris() {
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }
    
    public Aksesoris(String jenis, String bahan, String warna) {
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }
    
    public void setJenis(String jenis) {
        this.jenis = jenis;
    }
    
    public void setBahan(String bahan) {
        this.bahan = bahan;
    }
    
    public void setWarna(String warna) {
        this.warna = warna;
    }
    
    public String getJenis() {
        return this.jenis;
    }
    
    public String getBahan() {
        return this.bahan;
    }
    
    public String getWarna() {
        return this.warna;
    }
}

File: CPP/Aksesoris.cpp
#include "Petshop.cpp"

class Aksesoris : public Petshop
{
    private:
        string jenis;
        string bahan;
        string warna;

    public:
        Aksesoris(){
            this->jenis = "";
            this->bahan = "";
            this->warna = "";
        };
        
        Aksesoris(string jenis, string bahan, string warna)
        {
            this->jenis = jenis;
            this->bahan = bahan;
            this->warna = warna;
        }
        
        void setJenis(string jenis)
        {
            this->jenis = jenis;
        }

        void setBahan(string bahan)
        {
            this->bahan = bahan;
        }

        void setWarna(string warna)
        {
            this->warna = warna;
        }

        string getJenis()
        {
            return this->jenis;
        }

        string getBahan()
        {
            return this->bahan;
        }

        string getWarna()
        {
            return this->warna;
        }
        
        ~Aksesoris(){};
};
